.pipelines/load-tests-template.yaml

```diff
@@ -0,0 +1,125 @@
+jobs:
+  - job:
+    displayName: aks-load-test
+    timeoutInMinutes: 360
+    cancelTimeoutInMinutes: 30
+    workspace:
+      clean: all
+    variables:
+    - name: HOME
+      value: $(System.DefaultWorkingDirectory)
+    - name: GOPATH
+      value: $(HOME)/go
+    - name: perf-tests.repo.path
+      value: $(GOPATH)/src/k8s.io/perf-tests
+    - group: csi-secrets-store-load
+    steps:
+      - task: GoTool@0
+        inputs:
+          version: '1.14.1'
+
+      - bash: |
+          git clone -b $(checkout.branch) $(checkout.repo) $(perf-tests.repo.path)
```

Comment:
just a heads up that we are not pinning this since there are no tags for perf-tests

Reply:
Added this as a comment in the pipeline yaml